File: core/src/main/java/com/graphhopper/routing/weighting/custom/ParseResult.java
package com.graphhopper.routing.weighting.custom;

import java.util.Set;

public class ParseResult {
    StringBuilder converted;
    boolean ok;
    String invalidMessage;
    Set<String> guessedVariables;
    Set<String> operators;
    // the name and arguments of the built-in function call (see ValueExpressionVisitor.BUILT_INS), or null
    String function;
    String[] functionArgs;
}

File: core/src/main/kotlin/com/graphhopper/routing/weighting/custom/ParseResult.kt
package com.graphhopper.routing.weighting.custom

class ParseResult {
    @JvmField
    internal var converted: StringBuilder? = null

    @JvmField
    internal var ok = false

    @JvmField
    internal var invalidMessage: String? = null

    @JvmField
    internal var guessedVariables: MutableSet<String>? = null

    @JvmField
    internal var operators: MutableSet<String>? = null
}
